cicd/crabtaskworker_pypi/Dockerfile

```diff
@@ -17,11 +6,17 @@ RUN mkdir /build \
     && apt-get install -y curl zip git \
     && apt-get clean all
 WORKDIR /build
-COPY cicd/crabtaskworker_pypi/buildTWTarballs.sh cicd/crabtaskworker_pypi/buildDatafiles.sh /build/
+COPY cicd/crabtaskworker_pypi/buildTWTarballs.sh \
+     cicd/crabtaskworker_pypi/buildDatafiles.sh \
+     cicd/crabserver_pypi/wmcore_requirements.txt /build/
 COPY setup.py /build
 COPY src /build/src
 COPY scripts /build/scripts
-COPY --from=wmcore-src /WMCore /build/WMCore
+
+RUN wmcore_repo="$(grep -v '^\s*#' wmcore_requirements.txt | cut -d' ' -f1)" \
```

Comment:
does this mean that ` cicd/crabserver_pypi/wmcore_requirements.txt` needs to always specify a GH repository ? Forgive me for being a  bit lost. Do we have different needs in crabserver and crabtaskworker ?

Reply:
> does this mean that  cicd/crabserver_pypi/wmcore_requirements.txt needs to always specify a GH repository ?

Yes. 

> Do we have different needs in crabserver and crabtaskworker ?

Could you give me an example?

I think (did not check yet) when you package with `setup.py` in old rpm image, you will get different files in final installation path for crabserver and crabtaskworker.
But in current prod we installed the whole wmcore via pip, which should contain at least combination of files for crabserver and taskworker,  and it work just fine.